foundation-ui/services/AuthorizeBundle.js

```diff
@@ -0,0 +1,111 @@
+import React, {PropTypes} from 'react';
+
+import Hooks from '../services/Hooks';
+import ServiceFactory from '../factories/ServiceFactory';
+
+const hooks = new Hooks();
```

Comment:
This was a decision made in the design doc. Take a look at the "Formalizing Hook entry points" section in the design doc